src/ocean/util/container/map/model/StandardHash.d

```diff
@@ -80,7 +80,12 @@ struct StandardHash
                          ~ "classes/interfaces/structs/unions implementing "
                          ~ "toHash() supported, not '" ~ K.stringof ~ '\'');
 
-            return key.toHash();
+            // HACK: we don't want to start implementing all our class `toHash`
+            // methods as `const` yet, have to cast to silence druntime
+            static if (is(K == class) || is(K == interface))
+                return (cast(Unqual!(K)) key).toHash();
```

Comment:
this is D2 only right ?
please use `cast()` instead of `Unqual`

Reply:
No, this code compiles both in D1 and (after conversion) in D2. I don't think `cast()` works in D1.

Reply:
true it does not work in D1.
Maybe in a version(D_Version2) ?
It does cut down on unwanted template-instances.

Reply:
No.